<!DOCTYPE html>
<html><head><style>table, th, td {border: 1px solid black;border-collapse: collapse;}</style></head>
<table colsep="0" frame="topbot" id="tbl0005" trsep="0" xmlns="http://www.elsevier.com/xml/common/cals/dtd">
<label>
        Table 1
       </label>
<caption id="cap0015">
<simple-para id="spar0015" view="all">
         Main antimicrobial resistance genes found in Democratic Republic of the Congo.
        </simple-para>
</caption>

<tgroup cols="5">
<colspec colname="col1"/>
<colspec colname="col2"/>
<colspec colname="col3"/>
<colspec colname="col4"/>
<colspec colname="col5"/>
<thead valign="top">
<tr trsep="1">
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Species
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Methods
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           No. of strains
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Antibiotic resistance genes
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           References
          </td>
</tr>
</thead>
<tbody>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Salmonella
           </italic>
           spp
           <italic>
            .
           </italic>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           PCR, gene sequencing, whole-genome sequencing
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           453
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            bla
           </italic>
<inf loc="post">
            CTX-M-15
           </inf>
           ,
           <italic>
            bla
           </italic>
<inf loc="post">
            SHV-2a
           </inf>
           ,
           <italic>
            mph(A)
           </italic>
           ,
           <italic>
            gyrA
           </italic>
           mutation,
           <italic>
            qnrB1
           </italic>
           ,
           <italic>
            aac(6′)-Ib-cr
           </italic>
           ,
           <italic>
            bla
           </italic>
<inf loc="post">
            TEM-1
           </inf>
           ,
           <italic>
            bla
           </italic>
<inf loc="post">
            TEM-1b
           </inf>
           ,
           <italic>
            catA1
           </italic>
           ,
           <italic>
            strA/B
           </italic>
           ,
           <italic>
            sul1
           </italic>
           ,
           <italic>
            dfrA1
           </italic>
           ,
           <italic>
            aac(6′)-1aa
           </italic>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<cross-refs id="crfs0005" refid="bib0135 bib0185 bib0190 bib0200 bib0215 bib0225 bib0235 bib0385">
            [27,37,38,40,43,45,47,77]
           </cross-refs>
</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Escherichia coli
           </italic>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           PCR, whole-genome sequencing
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           21
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            bla
           </italic>
<inf loc="post">
            CTX-M-1
           </inf>
           ,
           <italic>
            bla
           </italic>
<inf loc="post">
            TEM
           </inf>
           ,
           <italic>
            bla
           </italic>
<inf loc="post">
            CMY
           </inf>
           ,
           <italic>
            bla
           </italic>
<inf loc="post">
            OXA
           </inf>
           ,
           <italic>
            aac3
           </italic>
           ,
           <italic>
            ant2′′
           </italic>
           ,
           <italic>
            ant3′′
           </italic>
           ,
           <italic>
            aph3′′
           </italic>
           ,
           <italic>
            aph6
           </italic>
           ,
           <italic>
            pbp2
           </italic>
           ,
           <italic>
            aac6′
           </italic>
           ,
           <italic>
            qnrB
           </italic>
           ,
           <italic>
            mphA
           </italic>
           ,
           <italic>
            cat
           </italic>
           ,
           <italic>
            catA
           </italic>
           ,
           <italic>
            catB
           </italic>
           ,
           <italic>
            arr
           </italic>
           ,
           <italic>
            folp
           </italic>
           ,
           <italic>
            sulI
           </italic>
           ,
           <italic>
            sulII
           </italic>
           ,
           <italic>
            tetA
           </italic>
           ,
           <italic>
            tetB
           </italic>
           ,
           <italic>
            tetD
           </italic>
           ,
           <italic>
            dhfr
           </italic>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<cross-refs id="crfs0010" refid="bib0265 bib0280">
            [53,56]
           </cross-refs>
</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Klebsiella
           </italic>
           spp
           <italic>
            .
           </italic>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           PCR
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           3
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            bla
           </italic>
<inf loc="post">
            CTX-M-15
           </inf>
           ,
           <italic>
            bla
           </italic>
<inf loc="post">
            SHV
           </inf>
           variants
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Staphylococcus aureus
           </italic>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           PCR, whole-genome sequencing
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           148
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            aac(6′)-aph(2′′)
           </italic>
           ,
           <italic>
            tetK
           </italic>
           ,
           <italic>
            emrC
           </italic>
           ,
           <italic>
            mecA
           </italic>
           gene found, mutation in
           <italic>
            femA
           </italic>
           gene found,
           <italic>
            dfrG
           </italic>
           ,
           <italic>
            rpoB
           </italic>
           ,
           <italic>
            emrC
           </italic>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<cross-refs id="crfs0015" refid="bib0140 bib0220">
            [28,44]
           </cross-refs>
</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd"/>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd"/>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd"/>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd"/>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd"/>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Vibrio cholerae
           </italic>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Whole-genome sequencing
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           78
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            cardB
           </italic>
           ,
           <italic>
            intSXT
           </italic>
           ,
           <italic>
            sxt/R391
           </italic>
           ,
           <italic>
            sulII
           </italic>
           ,
           <italic>
            sulI
           </italic>
           ,
           <italic>
            qnrVC
           </italic>
           ,
           <italic>
            aph3-dprime
           </italic>
           ,
           <italic>
            aph6
           </italic>
           ,
           <italic>
            drfA
           </italic>
           ,
           <italic>
            dhfr
           </italic>
           ,
           <italic>
            flor
           </italic>
           ,
           <italic>
            gyrA
           </italic>
           (S83I),
           <italic>
            parC
           </italic>
           (S85L)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
</tbody>
</tgroup>
</table>
</html><!DOCTYPE html>
<html><head><style>table, th, td {border: 1px solid black;border-collapse: collapse;}</style></head>
<table colsep="0" frame="topbot" id="tbl0010" trsep="0" xmlns="http://www.elsevier.com/xml/common/cals/dtd">
<label>
        Table 2
       </label>
<caption id="cap0020">
<simple-para id="spar0020" view="all">
         Prevalence of resistance among bacteria isolated in the Democratic Republic of the Congo.
        </simple-para>
</caption>

<tgroup cols="5">
<colspec colname="col1"/>
<colspec colname="col2"/>
<colspec colname="col3"/>
<colspec colname="col4"/>
<colspec colname="col5"/>
<thead valign="top">
<tr trsep="1">
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Study period
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Microorganism isolated (year of isolation)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Prevalence of resistance
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Gene identified
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           References
          </td>
</tr>
</thead>
<tbody>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           1993
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Salmonella
           </italic>
           spp. (1986)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           90% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           90) resistance to ampicillin, chloramphenicol, streptomycin, and tetracycline
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           ND
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2001
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Salmonella
           </italic>
           spp. and
           <italic>
            Escherichia coli
           </italic>
           (1990)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           100% resistance to ampicillin and chloramphenicol for
           <italic>
            Salmonella enteritidis
           </italic>
           , 65% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           46) resistance to ampicillin and chloramphenicol for
           <italic>
            E. coli
           </italic>
           (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           19)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           ND
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2009
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Salmonella
           </italic>
           Typhi (2005)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           All the strains (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           11) exhibited resistance to ampicillin (&gt;256
           <hsp sp="0.25"/>
           mg/L) and cotrimoxazole (&gt;32
           <hsp sp="0.25"/>
           mg/L)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           ND
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2010
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Salmonella
           </italic>
           spp. (2006)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           For
           <italic>
            S. typhimurium
           </italic>
           , high proportion of isolates were resistant to ampicillin (85.7%), chloramphenicol (92.5%), and cotrimoxazole (94.7%). 6.1% of
           <italic>
            S. enteritidis
           </italic>
           tested were resistant to nalidixic acid and ciprofloxacin. 10.2% of
           <italic>
            S. enteritidis
           </italic>
           isolates were resistant to gentamicin. 89.7% of
           <italic>
            S
           </italic>
           . Typhi isolates presented a similar antimicrobial-resistant profile as the one observed among NTS
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Two isolates displayed resistance to cefoxitin (32 mg/L); the presence of the genes
           <italic>
            bla
           </italic>
<inf loc="post">
            CMY-1
           </inf>
           ,
           <italic>
            bla
           </italic>
<inf loc="post">
            CMY-2
           </inf>
<italic>
            , bla
           </italic>
<inf loc="post">
            ACC
           </inf>
           ,
           <italic>
            bla
           </italic>
<inf loc="post">
            CTX-M
           </inf>
           , and
           <italic>
            bla
           </italic>
<inf loc="post">
            TEM
           </inf>
           was negative
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2013
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Salmonella typhimirium
           </italic>
           (2006)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           100% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           11) resistance to ampicillin, cotrimoxazole, and chloramphenicol
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            bla
           </italic>
<inf loc="post">
            TEM-1b
           </inf>
           ,
           <italic>
            catA1
           </italic>
           ,
           <italic>
            strA/B
           </italic>
           ,
           <italic>
            sul1
           </italic>
           ,
           <italic>
            dfrA1
           </italic>
           ,
           <italic>
            aac(6′)-1aa
           </italic>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2014
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Salmonella
           </italic>
           spp. (2011)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           96% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           180) resistance to ampicillin, chloramphenicol, and cotrimoxazole
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            bla
           </italic>
<inf loc="post">
            SHV-2a
           </inf>
           ,
           <italic>
            mph(A)
           </italic>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2014
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Salmonella
           </italic>
           spp. (2012)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           72.2% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           18) of
           <italic>
            S.
           </italic>
           Typhi isolates showed co-resistance to ampicillin and cotrimoxazole, 33.3% showed additional resistance to chloramphenicol and hence were MDR. 97.6% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           85) of NTS isolates were MDR; 1.0%
           <italic>
            S. typhimurium
           </italic>
           isolates had additional DCS
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            gyrA
           </italic>
           gene mutation found
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2015
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Salmonella
           </italic>
           spp. (2014)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           For
           <italic>
            S.
           </italic>
           Typhi isolates (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           164), MDR and DCS rates were 37.8% and 37.2%, respectively.
           <br/>
           MDR were 90.2% and 79.7%, respectively for
           <italic>
            S. typhimurium
           </italic>
           (386) and
           <italic>
            S. enteritidis
           </italic>
           (390). ESBL production was observed in 12.7% of
           <italic>
            S. typhimurium
           </italic>
           isolates
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            gyrA
           </italic>
           mutation found,
           <italic>
            qnrB1
           </italic>
           ,
           <italic>
            aac(6′)-Ib-cr
           </italic>
           ,
           <italic>
            bla
           </italic>
<inf loc="post">
            TEM-1
           </inf>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2017
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Salmonella
           </italic>
           Typhi (2015)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Production of ESBL in
           <italic>
            S.
           </italic>
           Typhi was confirmed by PCR for the first time in DRC. The isolate further showed decreased susceptibility to ciprofloxacin, to pefloxacin and to nalidixic acid.
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            bla
           </italic>
<inf loc="post">
            CTX-M-15
           </inf>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2018
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Salmonella
           </italic>
           Typhi (2017)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           ESBL-producing
           <italic>
            S.
           </italic>
           Typhi isolate with a decreased ciprofloxacin susceptibility
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            bla
           </italic>
<inf loc="post">
            CTX-M-15
           </inf>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2018
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Salmonella
           </italic>
           spp. (2014)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           27% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           60) resistance to cefuroxime, 20% to ceftriaxone and norfloxacin
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           ND
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2019
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Salmonella typhimurium
           </italic>
           (2016)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           67% (54/81) were ESBL producers and 63% were MDR (resistance to ampicillin, chloramphenicol, and cotrimoxazole).
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            catA
           </italic>
           ,
           <italic>
            bla
           </italic>
<inf loc="post">
            TEM-1
           </inf>
           ,
           <italic>
            dfrA
           </italic>
           ,
           <italic>
            dfrA1
           </italic>
           ,
           <italic>
            dfrA14
           </italic>
           ,
           <italic>
            bla
           </italic>
<inf loc="post">
            SHV-2A
           </inf>
           ,
           <italic>
            mphA
           </italic>
           ,
           <italic>
            gyrA
           </italic>
           mutation (S83Y)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2019
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Salmonella
           </italic>
           Typhi (2017)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           38.4% (114/295) were resistant to ampicillin, chloramphenicol, and cotrimoxazole; 24.5% (73/295) had decreased sensitivity to ciprofloxacin
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            gyrA
           </italic>
           (S83E, S83Y, D87Y, D87G, A119E),
           <italic>
            gyrB
           </italic>
           (E466D, S464Y)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2012
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Enterobacteriaceae (2011)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           7.4% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           190) ESBL-producing Enterobacteriaceae
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            bla
           </italic>
<inf loc="post">
            CTX-M-1
           </inf>
           group
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2014
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Enterobacteriaceae and non-fermenting Gram-negative rods (2013)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           16.3% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           643) of isolates displayed an MDR phenotype; 80% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           25) of isolates were ESBL producers
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            bla
           </italic>
<inf loc="post">
            CTX-M-1
           </inf>
           group
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2014
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Enterobacteriaceae (2013)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           100% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           79) resistance to tetracycline, over 90% resistance to ampicillin, cotrimoxazole, and chloramphenicol and over 65% resistance to norfloxacin
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           ND
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2015
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Enterobacteriaceae (2014)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           100% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           112) resistance to cotrimoxazole, 67–100% resistance to ampicillin and &gt;50% of them were ESBL producers
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           ND
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2015
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Enterobacteriaceae and Gram-positive cocci (2015)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           98% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           38) of
           <italic>
            E. coli
           </italic>
           and 65% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           22) of
           <italic>
            S. aureus
           </italic>
           resistant to penicillin, and 92% of
           <italic>
            E. coli
           </italic>
           and 73% of
           <italic>
            S. aureus
           </italic>
           resistant to cefotaxime
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           ND
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2012
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Klebsiella
           </italic>
           spp
           <italic>
            .
           </italic>
           and
           <italic>
            Enterobacter aerogenes
           </italic>
           (2010)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           5.3% (150) of ESBL producers (eight isolates).
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Five
           <italic>
            bla
           </italic>
<inf loc="post">
            CTX-M
           </inf>
           genes and three
           <italic>
            bla
           </italic>
<inf loc="post">
            SHV
           </inf>
           variants
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2019
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Escherichia coli
           </italic>
           (2014)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           All isolates were MDR, corresponding to ESBL producers (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           21). Twenty-one (21/21) and 19/21 strains were susceptible to imipenem and amikacin, respectively
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            bla
           </italic>
<inf loc="post">
            CTX-M-1
           </inf>
           ,
           <italic>
            bla
           </italic>
<inf loc="post">
            TEM
           </inf>
           ,
           <italic>
            bla
           </italic>
<inf loc="post">
            CMY
           </inf>
           ,
           <italic>
            bla
           </italic>
<inf loc="post">
            OXA
           </inf>
           ,
           <italic>
            aac3
           </italic>
           ,
           <italic>
            ant2′′
           </italic>
           ,
           <italic>
            ant3′′
           </italic>
           ,
           <italic>
            aph3′′
           </italic>
           ,
           <italic>
            aph6
           </italic>
           ,
           <italic>
            pbp2
           </italic>
           ,
           <italic>
            aac6′, qnrB, mphA
           </italic>
           ,
           <italic>
            cat
           </italic>
           ,
           <italic>
            catA, catB
           </italic>
           ,
           <italic>
            arr
           </italic>
           ,
           <italic>
            folP
           </italic>
           ,
           <italic>
            sulI
           </italic>
           ,
           <italic>
            sulII
           </italic>
           ,
           <italic>
            tetA
           </italic>
           ,
           <italic>
            tetB
           </italic>
           ,
           <italic>
            tetD
           </italic>
           ,
           <italic>
            dhfr
           </italic>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2014
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Staphylococcus
           </italic>
           spp
           <italic>
            .
           </italic>
           (2013)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           63.5% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           74) of
           <italic>
            S. aureus
           </italic>
           and 90% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           15) of CNS were MRSA and MR-CNS, respectively
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           ND
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2015
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Staphylococcus aureus
           </italic>
           (2011)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           16% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           63) were MRSA
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           ND
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2016
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Staphylococcus aureus
           </italic>
           (2014)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           33% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           100) were MRSA, 72% resistance to cyclins and 54% resistance to cotrimoxazole
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            mecA
           </italic>
           , mutation in
           <italic>
            femA
           </italic>
           gene found,
           <italic>
            dfrG
           </italic>
           ,
           <italic>
            rpoB
           </italic>
           ,
           <italic>
            emrC
           </italic>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2017
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Staphylococcus aureus
           </italic>
           (2016)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           36% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           45) of isolates were MRSA
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           ND
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2017
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Staphylococcus aureus
           </italic>
           (2013)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           25% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           108) were MRSA, 31% were resistant to aminoglycosides, 26% to macrolides, and 20% to ciprofloxacin
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            aac(6′)-aph(2′′)
           </italic>
           ,
           <italic>
            tetK
           </italic>
           ,
           <italic>
            emrC
           </italic>
</td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2015
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Vibrio cholerae
           </italic>
           (2013)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           98% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           36) resistance to cotrimoxazole, 71% to nitrofurantoin, and 18% to nalidixic acid.
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           ND
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2015
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Vibrio cholerae
           </italic>
           (2012)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           Although sensitivity to fluoroquinolones seemed to be preserved, strain resistance patterns continued to evolve with the circulation of isolates resistant to tetracyclines and ampicillin from 2007 to 2010, and displayed a single antimicrobial drug susceptibility profile: resistance to most antimicrobial drugs (chloramphenicol, erythromycin, cotrimoxazole) except cyclins and fluoroquinolones.
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           MLVA genotyping, no resistance genes searched
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2019
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Vibrio cholerae
           </italic>
           (2017)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           100% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           78) resistance and nalidixic acid, 12% and 14% resistance to ciprofloxacin and ampicillin, respectively
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            cardB
           </italic>
           ,
           <italic>
            IntSXT
           </italic>
           ,
           <italic>
            sxt/R391
           </italic>
           ,
           <italic>
            sulII
           </italic>
           ,
           <italic>
            sulI
           </italic>
           ,
           <italic>
            qnrVC
           </italic>
           ,
           <italic>
            aph3-dprime
           </italic>
           ,
           <italic>
            aph6
           </italic>
           ,
           <italic>
            drfA
           </italic>
           ,
           <italic>
            dhfr
           </italic>
           ,
           <italic>
            flor
           </italic>
           ,
           <italic>
            gyr
           </italic>
           A (S83I),
           <italic>
            parC
           </italic>
           (S85L)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2016
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Streptococcus pneumoniae and Haemophilus influenzae
           </italic>
           (2014)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           83% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           23) of
           <italic>
            S. pneumoniae
           </italic>
           isolates were resistant to penicillin, 28% (
           <italic>
            n
           </italic>
<hsp sp="0.25"/>
           =
           <hsp sp="0.25"/>
           23) of
           <italic>
            H. influenzae
           </italic>
           were resistant to amoxicillin–clavulanic acid
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           ND
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
<tr>
<td align="left" role="trhead" xmlns="http://www.elsevier.com/xml/common/dtd">
           2018
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
<italic>
            Streptococcus pneumoniae
           </italic>
           (2015)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           70of the 163 isolates (43%) of the pneumococci were MDR (non-susceptible to ≥3 classes of antimicrobial agents, including β-lactams).
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">
           ND (PCR done for serotyping only)
          </td>
<td align="left" xmlns="http://www.elsevier.com/xml/common/dtd">

</td>
</tr>
</tbody>
</tgroup>
<legend>
<simple-para id="spar0025" view="all">
         CNS, coagulase-negative staphylococcus; DCS, decreased ciprofloxacin susceptibility; DRC, Democratic Republic of the Congo; ESBL, extended-spectrum β-lactamase; MDR, multidrug resistant; MLVA, Multiple-locus variable number of tandem repeat analysis; MR-CNS, methicillin-resistant coagulase-negative staphylococci; MRSA, methicillin-resistant
         <italic>
          S. aureus
         </italic>
         ; ND, not determined; NTS, non-Typhi salmonella.
        </simple-para>
</legend>
</table>
</html>